rotkehlchen/tests/exchanges/test_bitstamp.py

```diff
@@ -0,0 +1,724 @@
+    """Test an entry that can't get its USD price is skipped
+    """
+    balances_data = (
+        """
+        {
+            "eth_available": "0.00000000",
+            "eth_balance": "32.00000000",
+            "eth_reserved": "0.00000000",
+            "eth_withdrawal_fee": "0.04000000",
+            "link_available": "0.00000000",
+            "link_balance": "1000.00000000",
+            "link_reserved": "0.00000000",
+            "link_withdrawal_fee": "0.25000000",
+            "xrp_available": "0.00000000",
+            "xrp_balance": "0.00000000",
+            "xrp_reserved": "0.00000000",
+            "xrp_withdrawal_fee": "0.02000000"
+        }
+        """
+    )
+
+    def mock_api_query_response(endpoint):  # pylint: disable=unused-argument
+        return MockResponse(HTTPStatus.OK, balances_data)
+
+    with patch.object(mock_bitstamp, '_api_query', side_effect=mock_api_query_response):
+        asset_balance, msg = mock_bitstamp.query_balances()
+        assert asset_balance == {
+            Asset('ETH'): Balance(
+                amount=FVal('32'),
+                usd_value=FVal('48'),
+            ),
+            Asset('LINK'): Balance(
+                amount=FVal('1000'),
+                usd_value=FVal('1500'),
+            ),
+        }
+        assert msg == ''
+
+
+# method: `_deserialize_trade()`
+def test_deserialize_trade__buy(mock_bitstamp):
+    raw_trade = {
+        'id': 2,
+        'type': 14,
+        'datetime': '2020-12-02 09:30:00',
+        'btc': '0.50000000',
+        'usd': '-10000.00000000',
+        'btc_usd': '0.00005000',
+        'fee': '20.00000000',
+        'order_id': 2,
+    }
+    expected_trade = Trade(
+        timestamp=1606901400,
+        location=Location.BITSTAMP,
+        pair=TradePair('BTC_USD'),
+        trade_type=TradeType.BUY,
+        amount=FVal("0.50000000"),
+        rate=FVal("0.00005000"),
+        fee=FVal("20.00000000"),
+        fee_currency=Asset('USD'),
+        link='2',
+        notes='',
+    )
+    trade = mock_bitstamp._deserialize_trade(raw_trade)
+    assert trade == expected_trade
+
+
+def test_deserialize_trade__sell(mock_bitstamp):
+    raw_trade = {
+        'id': 5,
+        'type': 14,
+        'datetime': '2020-12-03 11:30:00',
+        'eur': '-1.00000000',
+        'usd': '1.22000000',
+        'eur_usd': '0.81967213',
+        'fee': '0.00610000',
+        'order_id': 3,
+    }
+    expected_trade = Trade(
+        timestamp=1606995000,
+        location=Location.BITSTAMP,
+        pair=TradePair('EUR_USD'),
+        trade_type=TradeType.SELL,
+        amount=FVal("1.22000000"),
+        rate=FVal("0.81967213"),
+        fee=FVal("0.00610000"),
+        fee_currency=Asset('EUR'),
+        link='5',
+        notes='',
+    )
+    trade = mock_bitstamp._deserialize_trade(raw_trade)
+    assert trade == expected_trade
+
+
+# method: `_api_query_paginated()` for 'user_transactions' endpoint and trades
```

Comment:
Why are you adding these comments above the tests? It should be obvious from the test name which function is being tested. Don't add unnecessary comments